### PR TITLE
fix: improve v2 network support

### PR DESCRIPTION
**Pull request by JFrankfurt**

### CODE REVIEW

File: src/constants/chains.ts
Comment (tinaszheng) [inline src/constants/chains.ts:46]: sepolia works too!

**Review by tinaszheng — APPROVED** [review]
thoughts on also amending `useV2RouterContract` to return undefined if the chainId is not supported?

File: src/pages/Pool/index.tsx
Comment (zzmp) [inline src/pages/Pool/index.tsx:228]: My only issue here is that you're adding a new file to the V2 checking codepath, which complicates adding add'l chains. However, I don't think we plan on adding add'l chains to V2, so this seems fine.

**Review by zzmp — APPROVED** [review]

**Review by vm — APPROVED** [review]
just approving after main was merged

**Pull request merged**
